ui/js/controllers/perf/alerts.js

```diff
@@ -436,28 +435,27 @@ perf.controller('AlertsCtrl', [
             });
 
             $q.all(Object.keys(resultSetToSummaryMap).map(repo =>
-              ThResultSetModel.getResultSetList(
-                    repo, Object.keys(resultSetToSummaryMap[repo]), true).then(
-                        (response) => {
-                            response.data.results.forEach((resultSet) => {
-                                resultSet.dateStr = dateFilter(
-                                    resultSet.push_timestamp * 1000, thDateFormat);
-                                // want at least 14 days worth of results for relative comparisons
-                                const timeRange = phTimeRangeValues[repo] ? phTimeRangeValues[repo] : phDefaultTimeRangeValue;
-                                resultSet.timeRange = Math.max(timeRange,
-                                    phTimeRanges.map(timeRange => timeRange.value).find(
-                                    t => ((Date.now() / 1000.0) - resultSet.push_timestamp) < t));
-                                resultSetToSummaryMap[repo][resultSet.id].forEach(
-                                          (summary) => {
-                                              if (summary.push_id === resultSet.id) {
-                                                  summary.resultSetMetadata = resultSet;
-                                              } else if (summary.prev_push_id === resultSet.id) {
-                                                  summary.prevResultSetMetadata = resultSet;
-                                              }
-                                          });
-                            });
-
-                        }),
+              PushModel.getList({ id__in: Object.keys(resultSetToSummaryMap[repo]).join(',') })
+                  .then(async (response) => {
+                      const { results } = await response.json();
+                      results.forEach((resultSet) => {
+                          resultSet.dateStr = dateFilter(
```

Comment:
I realize it's still using the nomenclature of ``resultSet`` here.  But I figure these can be migrated when ``Perfherder`` itself is migrated to React to make diffing here easier.  But I'm happy to change them in this PR, if one of you desires it.  :)